backend/Squirrel.Core/Squirrel.Core.BLL/Services/ProjectService.cs

```diff
@@ -80,11 +80,17 @@ public async Task<ProjectResponseDto> GetProjectAsync(int projectId)
         var project = await _context.Projects
             .Include(project => project.Tags)
             .Include(project => project.Users)
+            .Include(project => project.Author)
```

Comment:
we have createdBy property, so you don't need to include author identity

Reply:
> we have createdBy property, so you don't need to include author identity

yeah, forgot about this)